discord/player.py

```diff
@@ -194,23 +194,31 @@ class FFmpegPCMAudio(FFmpegAudio):
         Extra command line arguments to pass to ffmpeg before the ``-i`` flag.
     options: Optional[:class:`str`]
         Extra command line arguments to pass to ffmpeg after the ``-i`` flag.
+    log_level: :class:`str`
+        The logging level to use for ffmpeg. Defaults to ``info``.
 
     Raises
     --------
     ClientException
         The subprocess failed to be created.
     """
 
-    def __init__(self, source, *, executable='ffmpeg', pipe=False, stderr=None, before_options=None, options=None):
-        args = []
+    def __init__(self, source, *, executable='ffmpeg', pipe=False, 
+                 stderr=None, before_options=None, options=None,
+                 log_level = "info"):
+        args = [
+            '-reconnect_at_eof', '1',
+            '-reconnect_delay_max', '4',
+            '-analyzeduration', '0',
+        ]
```

Comment:
These feel pretty arbitrary. I'm not sure if the order matters. I also feel like it should be controlled through a kw-only flag if someone doesn't want these flags to be added.